### DISCUSSION TITLE
sets of blocks?

### DISCUSSION THREAD

**Discussion by crswong888** [idaholab/moose#16548] [Q&A Meshing] [answered]
Is there a way to create a simple alias for a set of blocks? Most of the time I reference any of the individual blocks, its always the same ones grouped together, and they only need to be unique because of material properties. I'm dealing with a 3D model of a Westinghouse fuel assembly here, and so there's a lot going on.

As a simple example, I'm looking for some way to reference blocks 1 and 2 by a single name instead of both explicitly, while still being able to reference 1 and 2 individually:

```
[Variables]
  [disp_x]
    block = '1 2'
  []
[]

[Kernels]
  [some_kernel]
    block = '1 2'
  []
[]

[Materials]
  [mat1]
    block = 1
  []
  [mat2]
    block = 2
  []
[]
```

The model I'm working with has 18 blocks, not just 2, so trust me, it's a lot more painful than the above example.

**Comment by hugary1995** [comment, marked as answer]
How about defining
```
my_awesome_block_name = '1 2'
```
on the top of your input file?

Later on you can use
```
block = ${my_awesome_block_name}
```

**Reply by crswong888** [reply]
Wow I did not know this kind of syntax was available for MOOSE HIT files. This is very valuable. Thanks!

**Reply by aeslaughter** [reply]
https://mooseframework.inl.gov/application_usage/input_syntax.html
